src/commands/play.rs

```diff
@@ -64,6 +65,8 @@ pub async fn _play(
         EnqueueType::PLAYLIST => enqueue_playlist(&call, url).await,
     };
 
+    update_queue_messages(&ctx.http, &ctx.data, &call, guild_id).await;
```

Comment:
Again, I would update the queue at the end of this command, I think feedback to the command is important first.